cgi-bin/admin.c

```diff
@@ -1394,7 +1396,7 @@ do_config_server(http_t *http)		/* I - HTTP connection */
     if ((current_preserve_job_files = cupsGetOption("PreserveJobFiles",
                                                     num_settings,
 						    settings)) == NULL)
-      current_preserve_job_files = "1d";
+      current_preserve_job_files = "86400";
```

Comment:
Seems unnecessary - the cupsd.conf loader converts "1d" to 86400...

Reply:
Aha, note for myself - ```parse_variable()``` :) .